mk_verificator/tests/ceph/test_ceph_haproxy.py

```diff
@@ -0,0 +1,21 @@
+from mk_verificator import utils
+
+
+def test_ceph_haproxy(local_salt_client):
+    config = utils.get_configuration(__file__)
+
+    fail = {}
+
+    for monitor in config["ceph_monitors"]:
+        monitor_info = local_salt_client.cmd(monitor, 'cmd.run',
+                                             ["echo 'show stat' | nc -U "
+                                              "/var/run/haproxy/admin.sock | "
+                                              "grep ceph_mon_radosgw_cluster"])
+
+        for name, info in monitor_info.iteritems():
+            if "OPEN" and "UP" in info:
```

Comment:
yes we can replace it but if ceph mons will be changed we will edit the code but not the config. I will mark it as TODO in the file